addon/components/rl-week-picker.js

```diff
@@ -217,9 +219,11 @@ export default Ember.Component.extend(DropdownComponentMixin, {
   actions: {
     decreaseWeek: function () {
       if (!this.get('decreaseWeekButtonDisabled')) {
+        Ember.Logger.debug(this.currentMoment().format("LLL"));
         var m = this.currentMoment().subtract(1, 'weeks');
```

Comment:
I think maybe changing this line to:

```js
var m = this.currentMoment().isoWeekDay(4).subtract(1, 'weeks');
```
May solve the problem as well and then `if (newWeek === 52) { newYear -= 1; }` might not be necessary.

It's a really funny bug actually, because it seems like it depends on which day of the week you run the tests. If you create a moment from the week and year, it seems to set the weekday to the current weekday.